gammapy/background/tests/test_models.py

```diff
@@ -143,18 +248,35 @@ def test_read_write(self):
         assert_quantity_allclose(multi_array.counts.data, multi_array2.counts.data)
         assert_quantity_allclose(multi_array.livetime.data, multi_array2.livetime.data)
         assert_quantity_allclose(multi_array.bg_rate.data, multi_array2.bg_rate.data)
-        assert_quantity_allclose(multi_array.energy, multi_array2.energy)
-        assert_quantity_allclose(multi_array.offset, multi_array2.offset)
+        assert_quantity_allclose(multi_array.counts.energy, multi_array2.counts.energy)
+        assert_quantity_allclose(multi_array.counts.offset, multi_array2.counts.offset)
 
     def test_fillobs_and_computerate(self):
-        dir = str(gammapy_extra.dir) + '/datasets/hess-crab4-hd-hap-prod2'
-        data_store = DataStore.from_dir(dir)
-        obs_table = data_store.obs_table
-        multi_array = make_test_array()
-        multi_array.fill_obs(obs_table, data_store)
+        multi_array = make_test_array_fillobs()
         multi_array.compute_rate()
         assert_equal(multi_array.counts.data.value.sum(), 5403)
         pix = 23, 1
         assert_quantity_allclose(multi_array.livetime.data[pix], 6313.8117676 * u.s)
         rate = Quantity(0.0024697306536062276, "MeV-1 s-1 sr-1")
         assert_quantity_allclose(multi_array.bg_rate.data[pix], rate)
+
+    def test_fillobs_pie(self):
```

Comment:
@JouvinLea - I still have to prepare a presentation for tomorrow.

From my side, the most important points are:
- the high-level example that builds the background model should "work", i.e. the peaks at 0.5 and 1.5 deg in the acceptance curve should be gone
- there should be a test that covers the new functionality that passes on travis-ci.

As long as that's the case, I'm happy to merge if you want.
Of course, if you want a thorough code review and feedback, I can do that, but only tomorrow afternoon.

(I'll probably have to delay the Gammapy 0.4 release for a day or two because there's too much cleanup / outstanding little issues that are work in progress.)